packages/babel-plugin-transform-es2015-instanceof/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "babel-plugin-transform-es2015-instanceof",
   "version": "6.8.0",
-  "description": "",
+  "description": "Transforms all the ES2015 'instanceof' methods",
```

Comment:
To be consistent with the other descriptions maybe also start with `This plugin ...` or change the other plugins to start with `Transforms ...`. I'm more in favor of the later one, but it is also not that important. What do others think?